extensions/2.0/Khronos/KHR_spz_compression/README.md

```diff
@@ -0,0 +1,208 @@
+# KHR\_spz\_compression

```

Comment:
We need a different name for this. My current favorite is `KHR_gaussian_splatting_spz_compression` but it might be too long. The shorter `KHR_gsplat_spz_compression` might be better.

Reply:
We met with @lilleyse and decided on a name: `KHR_spz_gaussian_splats_compression`